py3status/modules/do_not_disturb.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-A simple "Do Not Disturb" module that can turn on and off all system notifications.
+Do not disturb by turning on and off all system notifications.
```

Comment:
Saw your question.
>Display is used many many times is this something needed?

I saw that too in the `list`. Crazy. Idk. We could fix that later.